### DISCUSSION TITLE
Wrong SSL read in browsers

### DISCUSSION THREAD

**Discussion by damms005** [FiloSottile/mkcert#438] [Q&A] [answered]
Strangely, I've been unable to get generated SSLs to work on my new computer (works on previous one). While trying to debug this, I found that the expiry dates reported by browsers for the cert are wrong. I will use snapshots of Chrome to explain.

After running `mkcert -install`, I generated the cert with `mkcert tims.local "*.tims.local" example.test localhost 127.0.0.1 ::1`. Command output was good.

I configured apache to use it. Relevant site config lines are:

```
        SSLEngine on
        SSLCertificateKeyFile   /etc/ssl/certs/tims.local+5-key.pem
        SSLCertificateFile      /etc/ssl/certs/tims.local+5.pem
````

Restarted apache, restarted browsers. Error persists. Clicking on "Not secure" on Chrome address bar area, I get this:
![image](https://user-images.githubusercontent.com/9839355/150081047-d8d31386-b012-4e31-94cd-89b4a370f345.png)

I ran the cert creation code at 
> Jan 19 2022 at 07:51
on my local computer. `ls /etc/ssl/certs/tims.local+5-key.pem` also confirms that.

Where did Chrome get an `Expires On` value of:
> Wednesday, 5 November 2031 at 02:15:54

My best guess is that somehow Chrome is holding on to some old, previously generated cert? I've tried clearing cache and all that, to no avail

**Comment by FiloSottile** [comment, marked as answer]
The certificate that is getting to the browser is not a mkcert certificate (mkcert certificate mention mkcert in the Organization field). That can be an issue with the web server configuration, or with the network connection not getting to the intended server, but it's not something mkcert can help with.
